### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by sk-t3ch** [sk-t3ch/smart-buoy#7] [Q&A]
For people who create their own smart-buoy!

**Comment by mriAppDev** [comment]
Greetings!

I've just come across your project and I'm really impressed and  intrigued.
Though I thought a Raspberry Pi was involved, it appears to an Arduino system (which is still very cool!).

Given the use of the solar panels for the system to be self-sufficient out at sea, I'm hopeful that I might be able to adapt this to power a RPi Zero for other projects.

While looking through the SUPPLIES section, it appears that all of the links (_except the 18650 Battery, which isn't available right now_) go to the same spot ==> 5 Pack Boost Converter Module XL6009

I know I've done this more than once myself, where I *think* I've got the link ready, but I somehow manage to PASTE the same thing more than once.  I'm hoping you can supply the links for the other items in the list -- **THANKS for doing this AND for putting all these resources up on GitHub!**

JC Irvine

![Snag_1f9fc19c](https://user-images.githubusercontent.com/5898510/124657302-56450780-de70-11eb-8d05-618fcbd3dca6.png)

**Comment by sk-t3ch** [comment]
Hello and welcome!

Thanks for the kind words. 
The Pi Zero may have low enough requirements for the system however it would still need to be secondarily power managed to turn on and off.

Thank you for letting us know about the links, we will fix them shortly. 

If you are interested in getting involved or viewing more of the project, we are in the process of writing a gitbook documentation similar to  https://docs.emlid.com/navio2/

Let us know :)
